libretro-common/include/libretro.h

```diff
@@ -976,6 +977,85 @@ struct retro_hw_render_context_negotiation_interface
                                             * so it will be used after SET_HW_RENDER, but before the context_reset callback.
                                             */
 
+/* Seek flags from stdio.h */
+enum retro_seek_flag
+{
+  RETRO_SEEK_SET = 0, /* Beginning of file */
+  RETRO_SEEK_CUR = 1, /* Current position of the file pointer */
+  RETRO_SEEK_END = 2, /* End of file - implementation not required */
+};
+
+/* Stat fields */
+struct retro_stat_struct
+{
+  uint32_t deviceId;         /* ID of device containing file */
```

Comment:
Are you sure this is needed? Symlink/hidden seem dubious as well, especially if there's no way to set them.